bundle/manifests/csi-driver-manila-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: csi-driver-manila-operator.v0.0.1
+  name: csi-driver-manila-operator.v4.5.0
```

Comment:
Shouldn't the file name be updated as well?

Reply:
yep :)